pkg/cover/report.go

```diff
@@ -88,12 +88,19 @@ func (rg *ReportGenerator) prepareFileMap(progs []Prog) (map[string]*file, error
 		}
 	}
 	progPCs := make(map[uint64]map[int]bool)
+	allProgPCs := make(map[uint64]bool)
+	unmatchedProgPCs := make(map[uint64]bool)
+	verifyCoverPoints := (len(rg.CoverPoints) > 0)
 	for i, prog := range progs {
 		for _, pc := range prog.PCs {
 			if progPCs[pc] == nil {
 				progPCs[pc] = make(map[int]bool)
 			}
 			progPCs[pc][i] = true
+			allProgPCs[pc] = true
```

Comment:
Nit: you don't need `allProgPCs` at all.

Reply:
Done.